lib/config.js

```diff
@@ -1,9 +1,14 @@
+var service_host = process.env.MONGO_SERVICE_HOST
+
 var database = {
-    url: 'mongodb://localhost:27017/pacman',
+    url: `mongodb://${service_host}:27017/pacman`,
```

Comment:
Can you help me understand how this helps you remove the HostPort requirement?

Are you setting the `MONGO_SERVICE_HOST` env var in your Dockerfile or perhaps in the pacman deployment k8s yaml resource? Can we set a default if the env var isn't defined like you've done for `MONGO_REPLICA_SET`?